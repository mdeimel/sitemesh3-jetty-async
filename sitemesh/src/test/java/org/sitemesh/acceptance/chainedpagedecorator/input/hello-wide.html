<html>
  <head>
    <title>Hello Wide</title>
  </head>
  <body>

   <p>Very simple document. Should appear in wide layout.</p>

  </body>
</html>