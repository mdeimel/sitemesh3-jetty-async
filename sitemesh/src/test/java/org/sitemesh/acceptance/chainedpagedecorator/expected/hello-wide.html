<html>
  <head>
    <title>Hello Wide</title>
    <style type="text/css">
      .wide { padding: 0; }
    </style>
    <style type="text/css">
      .main { background-color: #eee; }
      body { font-family: arial,serif; }
    </style>
  </head>
  <body>

    <h1>Hello Wide</h1>

    <div class="main">
      <div class="wide">
        <p>Very simple document. Should appear in wide layout.</p>
      </div>
    </div>

  </body>
</html>